### PR TITLE
Add LED-GPIO support using for QUANTA IX1B switch

### PR DESCRIPTION
**Pull request by cpchangtw**
Add led-gpio kernel module which used for controlling status led on front panel.

Signed-off-by: Chih-Pei Chang <Chih-Pei.Chang@qct.io>

### CODE REVIEW

File: patch/config-quanta-ix1b.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+old mode 100644
+new mode 100755
```
Comment (lguohan) [inline patch/config-quanta-ix1b.patch]: do not change the file mode

Comment (cpchangtw) [reply patch/config-quanta-ix1b.patch]: commit again, sorry for mistake.

File: patch/config-quanta-ix1b.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+index 3d31fb2..97827d4 100644
+--- a/debian/build/build_amd64_none_amd64/.config
++++ b/debian/build/build_amd64_none_amd64/.config
+@@ -5128,6 +5128,7 @@ CONFIG_LEDS_CLASS=y
+ CONFIG_LEDS_PCA9532=m
+ # CONFIG_LEDS_PCA9532_GPIO is not set
+ # CONFIG_LEDS_GPIO is not set
```
Comment (lguohan) [inline patch/config-quanta-ix1b.patch]: remove CONFIG_LEDS_GPIO is not set

scripts/kconfig/conf --silentoldconfig Kconfig
.config:5132:warning: override: reassigning to symbol LEDS_GPIO

Comment (cpchangtw) [reply patch/config-quanta-ix1b.patch]: already removed the line, please checked, thanks.

**Comment by lguohan** [comment]
@cpchangtw, can you check your config patch. does not look correct.

error: patch failed: debian/build/build_amd64_none_amd64/.config:5128
error: debian/build/build_amd64_none_amd64/.config: patch does not apply
stg import: Diff does not apply cleanly

**Pull request merged**